### PR TITLE
Update Our Color Palette to the Most Recent Version

### PR DESCRIPTION
**Pull request by porada**
* Upgrade `@automattic/color-studio` to the most recent version (that updates to Blue and Celadon: Automattic/color-studio#37).
* Tweak the SVG audit script to properly catch all images that might use non-standard color values.
* Update affected images so that they only use standardized color values.
* Update any outdated static color values found in the codebase.

#### Testing Instructions

*Local Testing Only*

* Pull the changes:
    * `git checkout update/colors`
    * `yarn`
* Run the SVG color audit script and make sure it doesn’t report any specific paths.
    * `node bin/audit-svg-colors`
* Preview the images listed by the command below:
    * `git diff trunk --name-only | grep 'svg$'`
    * Make sure none of them exhibits any visual regressions or errors that would prevent them from displaying.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -161,7 +161,7 @@
 		"@automattic/calypso-color-schemes": "^2.1.1",
 		"@automattic/calypso-doctor": "^0.1.0",
 		"@automattic/calypso-polyfills": "^1.0.0",
-		"@automattic/color-studio": "^2.3.0",
+		"@automattic/color-studio": "2.4.0",
```
Comment (porada) [inline package.json:164]: Intentionally omitted the `^`.

**Review by sirbrillig — APPROVED** [review]
I can't speak to the other parts, but approving this from a checkout perspective!

**Comment by porada** [comment]
@sirbrillig — thanks for reviewing!

Just rebased to resolve conflicts.

@frontdevde — do you think this can PR be merged right after a passing review from you?

**Comment by frontdevde** [comment]
No objections from my end! 👍 We'll have to update some of the color schemes on the wp-admin side for which I'll try to use color studio so we have the same dependency on both ends. I'll be probably working on it this week. This shouldn't block this PR though as Nav Unification isn't live for customers yet.

**Comment by porada** [comment]
Awesome. Thanks!

The failing E2E tests aren’t related to this PR, so I’m merging this…

**Pull request merged**